Panic when calling a JS function from a Go function

I'm attempting to pass a JS function as a parameter to a Go callback, then later invoke the JS function using its Call method, resulting in the error shown below. I may be doing this incorrectly, or I may be wrong to expect it to work at all.

Using the latest v8go (v0.5.2-0.20210416054929-571ffb097524) as of April 20th.

```go
package main

import (
	"fmt"

	"rogchap.com/v8go"
)

func main() {
	iso, _ := v8go.NewIsolate()
	gbl, _ := v8go.NewObjectTemplate(iso)
	doCallback, _ := v8go.NewFunctionTemplate(iso, func(info *v8go.FunctionCallbackInfo) *v8go.Value {
		tf, _ := info.Args()[0].AsFunction()
		tf.Call()
		return nil
	})
	gbl.Set("doCallback", doCallback)
	ctx, _ := v8go.NewContext(iso, gbl)
	val, err := ctx.RunScript("doCallback(() => true)", "test.js")
	fmt.Printf("%v %v", val, err)
}
```

```sh
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x86d2d6]

goroutine 1 [running]:
rogchap.com/v8go.(*Context).register(0x0)
	/mnt/storage/home/echicken/Documents/Development/go/src/github.com/echicken/callbacks/vendor/rogchap.com/v8go/context.go:129 +0x36
rogchap.com/v8go.(*Function).Call(0xc000010040, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/mnt/storage/home/echicken/Documents/Development/go/src/github.com/echicken/callbacks/vendor/rogchap.com/v8go/function.go:28 +0x21a
main.main.func1(0xc000064020, 0x0)
	/mnt/storage/home/echicken/Documents/Development/go/src/github.com/echicken/callbacks/main.go:17 +0x148
rogchap.com/v8go.goFunctionCallback(0x1, 0x1, 0x7ffe1323cbd0, 0x1, 0x0)
	/mnt/storage/home/echicken/Documents/Development/go/src/github.com/echicken/callbacks/vendor/rogchap.com/v8go/function_template.go:77 +0x2fd
rogchap.com/v8go._Cfunc_RunScript(0x366b430, 0x366b410, 0x366b3f0, 0x0, 0x0, 0x0, 0x0)
	_cgo_gotypes.go:668 +0x50
rogchap.com/v8go.(*Context).RunScript.func3(0xc00000e030, 0x366b410, 0x366b3f0, 0x0, 0x0, 0x0, 0x0)
	/mnt/storage/home/echicken/Documents/Development/go/src/github.com/echicken/callbacks/vendor/rogchap.com/v8go/context.go:101 +0xad
rogchap.com/v8go.(*Context).RunScript(0xc00000e030, 0x140cbe5, 0x16, 0x140a283, 0x7, 0x0, 0x0, 0x0)
	/mnt/storage/home/echicken/Documents/Development/go/src/github.com/echicken/callbacks/vendor/rogchap.com/v8go/context.go:101 +0x173
main.main()
	/mnt/storage/home/echicken/Documents/Development/go/src/github.com/echicken/callbacks/main.go:22 +0x248
```

--- a/function_template.go
+++ b/function_template.go
@@ -69,7 +69,7 @@ func goFunctionCallback(ctxref int, cbref int, args *C.ValuePtr, argsCount int) 
 
 	argv := (*[1 << 30]C.ValuePtr)(unsafe.Pointer(args))[:argsCount:argsCount]
 	for i, v := range argv {
-		val := &Value{ptr: v}
+		val := &Value{ptr: v, ctx: ctx}
 		info.args[i] = val
 	}
 

